framework/src/actions/CreateProblemDefaultAction.C

```diff
@@ -76,7 +76,7 @@ CreateProblemDefaultAction::act()
 
       params.set<MooseMesh *>("mesh") = _mesh.get();
       params.set<bool>("use_nonlinear") = _app.useNonlinear();
-      if (_pars.isParamSetByUser("_solve"))
+      if (_pars.isParamValid("_solve"))
```

Comment:
this used to say true for setting it in a unit test. 
But it s a suppressed param, so it was not really set by a user, it was set programmatically "as if a user" but through an API a user does not have access to. 

anyway this works now